tests/serverpod_test_server/test_integration/test_tools/database_operations_test.dart

```diff
@@ -319,6 +319,7 @@ void main() {
           });
         },
         rollbackDatabase: RollbackDatabase.disabled,
+        testGroupTagsOverride: ['concurrency_one'],
```

Comment:
Suggestion: Break this out into a constant that is shared.

Reply:
I know this value doesn't really matter, but it might in the future.

Reply:
Ah shoot, good catch 🙏  I actually had this on my TODO list 😅  Doesn't help though if you don't look at the list...

Reply:
68757e6